File: pinyougou-parent/pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.item.Item;
import cn.itcast.core.pojo.seckill.SeckillGoods;

import java.util.List;

public interface SeckillService {


    List<Item> findItemById(Long goodsId);


    void applySeckill(SeckillGoods seckillGoods, Long[] selectItemIds);

    List<SeckillGoods> findList();

    SeckillGoods findOneFromRedis(Long id);

    void submitOrder(Long seckillId, String userId);
}

File: pinyougou-parent/pinyougou-web-shop/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.applySeckill=function(seckill,selectItemIds){
		return $http.post('../seckill/applySeckill.do?selectItemIds='+selectItemIds,seckill);
	}

    this.findItemById = function (id) {
		return $http.get('../seckill/findItemById.do?goodsId='+id);
    }

});
